fails if scripts dir is a git repo

My scripts dir is a git repo for other employees to be able to submit scripts to. Reload fails on the .git folder within the scripts folder.

`Hubot reloader: { [Error: Cannot find module 'C:\myhubot\scripts\.git'] code: 'MODULE_NOT_FOUND' }`

--- a/src/reload-scripts.coffee
+++ b/src/reload-scripts.coffee
@@ -41,7 +41,7 @@ success = (msg) ->
 # ref: https://github.com/srobroek/hubot/blob/e543dff46fba9e435a352e6debe5cf210e40f860/src/robot.coffee
 deleteScriptCache = (scriptsBaseDir) ->
   if Fs.existsSync(scriptsBaseDir)
-    for file in Fs.readdirSync(scriptsBaseDir).sort()
+    for file in Fs.readdirSync(scriptsBaseDir).sort() when file.substring(0,1) != '.'
       full = Path.join scriptsBaseDir, file
       if require.cache[require.resolve(full)]
         try

